### PR TITLE
Preview blog/doc for dedicated data-endpoints.

### PR DESCRIPTION
**Pull request by SteveLasker**
Signed-off-by: Steve Lasker <stevelasker@hotmail.com>

### CODE REVIEW

**Comment by SteveLasker** [comment]
Review please:
@yugangw-msft @sajayantony

File: docs/dedicated-data-endpoints/dedicated-data-endpoints.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Azure Container Registry: Dedicated Data Endpoints – Mitigating Data Exfiltration
+
+Azure Container Registry announces dedicated data-endpoints, enabling client firewall rules on registries, minimizing data exfiltration concerns.
```
Comment (sajayantony) [inline docs/dedicated-data-endpoints/dedicated-data-endpoints.md:3]: "enabling client firewall rules on registries" sounds like the registry is supporting "client firewall  rules". Shouldn't this be something like enabling customers to create more scoped client firewall rules in their environments?

File: docs/dedicated-data-endpoints/dedicated-data-endpoints.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+Using [az cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) version 2.3.1 or greater, run the [az acr update](https://docs.microsoft.com/en-us/cli/azure/acr?view=azure-cli-latest#az-acr-update) command:
+
+```sh
+az acr update --name contoso --data-endpoint-enabled
+```
+
+To view the data-endpoints, including regional endpoints for geo-replicated registries, use the az acr show-endpoints cli:
+
+```sh
```
Comment (sajayantony) [inline docs/dedicated-data-endpoints/dedicated-data-endpoints.md:80]: @yugangw-msft is this released already. Did you review the API already?

**Comment by SteveLasker** [comment]
@dlepow :eyes: please

File: docs/dedicated-data-endpoints/dedicated-data-endpoints.md (hunk 2 above)
Comment (SteveLasker) [reply docs/dedicated-data-endpoints/dedicated-data-endpoints.md:96]: This was changed to reflect what @yugangw-msft provided

File: docs/dedicated-data-endpoints/dedicated-data-endpoints.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+<img src="./media/registry-dual-endpoints.png" width=500x>
+
+## Registry Managed Storage Accounts
+
+Azure Container Registry is a multi-tenant service, where the data-endpoint storage accounts are managed by the registry service. There are many benefits for managed storage, such as load balancing, contentious content splitting, multiple copies for higher concurrent content delivery, and multi-region support with [geo-replication](https://aka.ms/acr/geo-replicatin).
+
+## Azure Private Link VNet Support
+
+Azure Container Registry recently announced [Private Link support](https://aka.ms/acr/privatelink), enabling a private endpoints from Azure VNets to be placed on the managed registry service. In this case, both the registry and data-endpoints are accessible from within the VNet, using private IPs. 
+
```
Comment (dlepow) [inline docs/dedicated-data-endpoints/dedicated-data-endpoints.md:20]: ...enabling a private **endpoint** ...

File: docs/dedicated-data-endpoints/dedicated-data-endpoints.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+Unfortunately, VNet connectivity isn’t always an option.
+
+## Client Firewall Rules & Data Exfiltration Risks
+
+When connecting to a registry from on-prem hosts, IoT devices, custom build agents, or when Private Link may not be an option, client firewall rules may be applied, limiting access to specific resources.
+
+<img src="./media/registry-client-rules-all-storage.png" width=500x>
+
+As customers locked down their client firewall configurations, they realized they must create a rule with a wildcard for all storage accounts, raising concerns for data-exfiltration. A bad actor could deploy code that would be capable of writing to another "Evil-Empire" storage account.
+
```
Comment (dlepow) [inline docs/dedicated-data-endpoints/dedicated-data-endpoints.md:34]: Not sure about term "Evil-Empire" here - didn't that refer to MSFT :). I see it in your diagram, perhaps substitute something like "Bad Actor"

File: docs/dedicated-data-endpoints/dedicated-data-endpoints.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  "dataEndpointEnabled": true,
+      "dataEndpointHostNames": [
+        "demo42.eastus.data.azurecr.io",
+        "demo42.westus.data.azurecr.io"
+  ```
+
+### az CLI
+
+Using [az cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) version 2.3.1 or greater, run the [az acr update](https://docs.microsoft.com/en-us/cli/azure/acr?view=azure-cli-latest#az-acr-update) command:
+
```
Comment (dlepow) [inline docs/dedicated-data-endpoints/dedicated-data-endpoints.md:89]: CLI version OK? I didn't find the `--data-endpoint-enabled` in my Mac 2.3.1 installation.

**Pull request merged**